clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -706,10 +706,10 @@ private KafkaConsumer(ConsumerConfig config,
                   Metrics metrics,
                   SubscriptionState subscriptions,
                   Metadata metadata,
-                  boolean autoCommitEnabled,
                   int autoCommitIntervalMs,
                   int heartbeatIntervalMs,
                   long retryBackoffMs,
+                  boolean autoCommitEnabled,
```

Comment:
nit: the previous location seemed a little better since it was listed next to `autoCommitIntervalMs`.